awx/api/serializers.py

```diff
@@ -4536,67 +4533,41 @@ def validate(self, attrs):
         return accepted
 
 
-class BulkJobNodeSerializer(serializers.Serializer):
-    # We don't do a PrimaryKeyRelatedField for unified_job_template and inventory, because that increases the number
+class BulkJobNodeSerializer(WorkflowJobNodeSerializer):
```

Comment:
The idea is that `WorkflowJobNodeSerializer` gives us automatic definition of all the `char_prompts` fields, which are not real fields, but have specific validation rules, like the choices for verbosity. We have a lot of friction modifying prompts (just last release...) so this is why I want these inherited.